bugbug/repository.py

```diff
@@ -41,6 +41,15 @@ def _init(repo_dir):
 
 
 def _transform(commit):
+    def is_test(p):
```

Comment:
Move the function outside of _transform, rename `p` to `path`.
Add a link to the source code where you got the function.

Reply:
The second part of the comment still applies

Reply:
I have added the link in the commit message:

>  Added function is_test to check whether a path is related to test or not. The conditions for is_test were obtained from https://github.com/mozsearch/mozsearch/blob/2e24a308bf66b4c149683bfeb4ceeea3b250009a/router/router.py#L127

Should I also add it as a comment in the code?

Reply:
Yes, just add a comment saying
# This code was adapted from XXX